GNUmakefile

```diff
@@ -54,7 +68,7 @@ test-compile:
 		echo "  make test-compile TEST=./$(PKG_NAME)"; \
 		exit 1; \
 	fi
-	go test -c $(TEST) $(TESTARGS)
+	go test -tags ALL -c $(TEST) $(TESTARGS)
```

Comment:
Might this be a mistake? Is this running the whole test suite in `test-compile`, while it should just run compilation + unit tests?

Reply:
Indeed. I ported the assumption from `go-vcloud-director`.
Simplified and fixed